main.go

```diff
@@ -1076,6 +1086,158 @@ func main() {
 		return Redirect(fmt.Sprintf("/users/%s/books/%s", user.Slug, vars["isbn"]))
 	}, loggedinMiddleware)
 
+	POST("/users/export/{user}", func(w Response, r Request) Output {
+		vars := VARS(r)
+
+		user, err := Q.UserBySlug(r.Context(), vars["user"])
+		if err != nil {
+			return NotFound
+		}
```

Comment:
we have to make sure that the current user is the same as `user` otherwise one user can export data for another user. to do that check other functions user the `can` function